memcache_proxy/dtog_test.go

```diff
@@ -749,6 +749,7 @@ func TestAll(t *testing.T) {
 				"STAT curr_items 444\r\n",
 				"STAT bytes 555\r\n",
 				"STAT oldest_item_age 666\r\n",
+				"STAT version not implemented\r\n",
```

Comment:
we decided on "not implemented" last time, I think we should stick with that.  I can turn it into a constant though, I like that 👍